coreservices/useradministration/src/CatenaX.NetworkServices.UserAdministration.Service/appsettings.Development.json

```diff
@@ -6,4 +6,4 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   }
-}
+}
```

Comment:
Please make sure not to commit any changes to the appsettings.Development.json

Reply:
accidentally committed the changes and later removed.